subprojects/shipkit/src/test/groovy/org/shipkit/internal/notes/contributors/IgnoredContributorTest.groovy

```diff
@@ -0,0 +1,30 @@
+package org.shipkit.internal.notes.contributors
+
+import spock.lang.Specification
+
+class IgnoredContributorTest extends Specification {
+
+    def "should ignore contributor from ignored list"() {
+        def ignoredContributor = IgnoredContributor.of(["ignoredContributor"])
+        def contributor = new DefaultProjectContributor("name", "ignoredContributor", "profileUrl", 1)
+
+        expect:
+        ignoredContributor.isTrue(contributor) == true
```

Comment:
You can just use `ignoredContributor.isTrue` or `!ignoredContributor.isTrue` instead of comparing to boolean literals.